config/install/oe_corporate_blocks.footer_link.general.about_us.yml

```diff
@@ -0,0 +1,8 @@
+langcode: en
+status: true
+dependencies: {  }
+id: about_us
+label: 'About us'
```

Comment:
Translations for this footer link are not working (neither the label nor the link), they are present tho. Might be only on my local but please take a look.

Reply:
For me is translated:
![image](https://github.com/user-attachments/assets/ac6143ca-f93d-411f-b80a-79610a7bfc97)